hack/make-rules/test-e2e-node.sh

```diff
@@ -115,6 +115,10 @@ if [[ ${run_until_failure} == "true" ]]; then
   ginkgoflags="${ginkgoflags} --until-it-fails=true "
 fi
 
+if [[ "${container_runtime_endpoint}" =~ /containerd.sock$ ]]; then
+  test_args+=" --kubelet-flags=\"--runtime-cgroups=/system.slice/containerd.service\" ${test_args}"
```

Comment:
There was a space required here as it is the first time test_args is appended by the script.